src/block.js

```diff
@@ -14,8 +28,7 @@ router.get('/header', async (req, res) => {
 
 // get current
 router.get('/current', async (req, res) => {
-    const raw = req.query['raw'] === 'true';
```

Comment:
Is this param useless? `const raw = req.query['raw'] === 'true';`

Reply:
https://electrumx.readthedocs.io/en/latest/protocol-methods.html#blockchain-headers-subscribe

As i see method blockchain.headers.subscribe() doesn't need a parameter